projects/kit/components/input-date-time/input-date-time.component.ts

```diff
@@ -312,7 +312,7 @@ export class TuiInputDateTimeComponent
         if (
             this.value[0] === null ||
             this.value[1] !== null ||
-            this.nativeValue.length <= this.fillerLength + DATE_TIME_SEPARATOR.length ||
+            this.nativeValue.length === this.fillerLength ||
```

Comment:
@alyonagerasimova 
Thanks for your contribution!

* Could you add playwright tests for this case inside [this file](https://github.com/taiga-family/taiga-ui/blob/v3.x/projects/demo-playwright/tests/kit/input-date-time/input-date-time.spec.ts), please ?
  ```ts
  test('Good description', async ({page}) => {
      await tuiGoto(page, '/components/input-date-time/API?timeMode=HH:MM:SS.MSS');
  
      await inputDateTime.textfield.fill('07.06.2024, 23:59');
      await inputDateTime.textfield.blur();
  
      await expect(inputDateTime.textfield).toHaveValue('07.06.2024, 23:59:00.000');
  });
  ```

* Cherry-pick your fix to `main` branch too, please

Reply:
I see you close your previous PR to `main`:
* https://github.com/taiga-family/taiga-ui/pull/7656

Sorry for confusion.
Let's me clarify it.
We require this fix in both `v3.x` and `main` branches (2 PR in total).

Reply:
Yes, I prematurely deleted my first fork repository in order to fork it again. I will open a new PR in the main branch

Reply:
https://github.com/taiga-family/taiga-ui/pull/7669